app/utils/details-page-title-factory.js

```diff
@@ -0,0 +1 @@
+export {default, initialize} from 'ember-frost-page-title/utils/details-page-title-factory'
```

Comment:
What is the `initialize` content going to be?  I do not see this named export in _addon/utils/details-page-title-factory.js_

Reply:
oops - thanks.